features/get_collections.feature

```diff
@@ -87,3 +87,51 @@ Feature: Get Collections
                 "errors":[ {"message":  "invalid order_by"}]
             }
             """
+
+    Scenario: GET /collections with name search
```

Comment:
Should there be a scenario to bring back more than one collection based on name search (filtering by name)?